app/views/users/show.html.haml

```diff
@@ -0,0 +1,36 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest('knwang@gmail.com'.downcase)}?s=40")
+          %h2 #{@user.name}'s video collections (#{@user.queue_items.count})
+          %button.btn.btn-default= link_to "Follow", relationships_path(leader_id: @user.id), method: :post if current_user.can_follow?(@user)
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queue_items.each do |queue_item|
+              %tr
+                %td
+                  = link_to queue_item.video_title, video_path(queue_item.video)
+                %td
+                  = link_to queue_item.category_name, category_path(queue_item.category)
```

Comment:
Instead of `category_path(queue_item.category)` you could use short-hand routes `queue_item.category`